internal/oci/notation/notation_test.go

```
@@ -0,0 +1,804 @@
+	// doRequests issues n authenticated GET requests through the given client,
+	// setting the scope hint on the context exactly like the ORAS repository
+	// methods do during verification.
+	doRequests := func(client *oauth.Client, n int) {
+		for i := 0; i < n; i++ {
+			ctx := oauth.WithScopesForHost(context.Background(), host, scope)
+			req, err := http.NewRequestWithContext(ctx, http.MethodGet,
+				fmt.Sprintf("%s/v2/%s/manifests/latest", baseURL, repository), nil)
+			g.Expect(err).NotTo(HaveOccurred())
+			resp, err := client.Do(req)
+			g.Expect(err).NotTo(HaveOccurred())
+			resp.Body.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)